internal/parser/parser/pptx_tcadp.go

```
@@ -1,95 +1,14 @@
-
-	models "ragflow/internal/entity/models"
-)
+import "context"
 
 // parsePresentationWithTCADP sends binary presentation (PPTX/PPT) data
-// to the TCADP cloud reconstruction service and returns the structured
-// parse result. Mirrors the spreadsheet-family parseSpreadsheetWithTCADP
-// in xls_tcadp.go
+// to the TCADP cloud reconstruction service. Thin wrapper over the shared
+// parseWithTCADP core — env-fallbacks and request construction live there.
 func parsePresentationWithTCADP(ctx context.Context, filename string, data []byte, fileType string,
```

golangci-lint (2.12.2)

[error] func parsePresentationWithTCADP is unused

(unused)